docs/devGuide/settingUp.md

```diff
@@ -22,11 +22,17 @@ This page explains how to set up your development environment to start contribut
    **Graphviz** v2.38 or later<br>
    %%(The above two are required for one of the third-party libraries used by MarkBind)%%
 
+1. **Python** 3 or later, and<br>
+   **pre-commit** v2.20 or later<br>
+   %%(The above two are required for python pre-commit hooks)%%
```

Comment:
Should we be using the `.pyz` instead? I think some might not be comfortable installing a global dependency.

Reply:
Sounds like a good idea and that will reduce the dependency to just python.